compatibility-verifier/sample-test-suite/config/query-results/feature-test-1-rest-sql.results

```diff
@@ -21,7 +21,7 @@
 {"resultTable":{"dataSchema":{"columnDataTypes":["DOUBLE","DOUBLE","DOUBLE","DOUBLE","DOUBLE","DOUBLE"],"columnNames":["sum(intMetric1)","summv(intDimMV1)","min(intMetric1)","minmv(intDimMV2)","max(longDimSV1)","maxmv(intDimMV1)"]},"rows":[[4.294967536E9,-2.147479976E9,0.0,6.0,7611.0,462.0]]},"exceptions":[],"numServersQueried":1,"numServersResponded":1,"numSegmentsQueried":1,"numSegmentsProcessed":1,"numSegmentsMatched":1,"numDocsScanned":10,"numEntriesScannedPostFilter":40,"numGroupsLimitReached":false,"totalDocs":10,"timeUsedMs":5,"segmentStatistics":[],"traceInfo":{},"minConsumingFreshnessTimeMs":0}
 {"resultTable":{"dataSchema":{"columnDataTypes":["LONG","LONG","DOUBLE","DOUBLE","DOUBLE","DOUBLE"],"columnNames":["count(*)","countmv(intDimMV1)","avg(floatMetric1)","avgmv(intDimMV2)","minmaxrange(doubleMetric1)","minmaxrangemv(intDimMV2)"]},"rows":[[10,19,114.09000263214111,1516.9,250.00000000000003,6656.0]]},"exceptions":[],"numServersQueried":1,"numServersResponded":1,"numSegmentsQueried":1,"numSegmentsProcessed":1,"numSegmentsMatched":1,"numDocsScanned":10,"numEntriesScannedPostFilter":40,"numGroupsLimitReached":false,"totalDocs":10,"timeUsedMs":7,"segmentStatistics":[],"traceInfo":{},"minConsumingFreshnessTimeMs":0}
-{"resultTable":{"dataSchema":{"columnDataTypes":["DOUBLE","DOUBLE","LONG","LONG","DOUBLE","DOUBLE"],"columnNames":["percentile(longDimSV1, 80.0)","percentilemv(intDimMV1, 90.0)","percentileest(longDimSV1, 80.0)","percentileestmv(intDimMV1, 90.0)","percentiletdigest(longDimSV1, 80.0)","percentiletdigestmv(intDimMV1, 90.0)"]},"rows":[[7611.0,462.0,7611,462,7611.0,462.0]]},"exceptions":[],"numServersQueried":1,"numServersResponded":1,"numSegmentsQueried":1,"numSegmentsProcessed":1,"numSegmentsMatched":1,"numDocsScanned":10,"numEntriesScannedPostFilter":20,"numGroupsLimitReached":false,"totalDocs":10,"timeUsedMs":8,"segmentStatistics":[],"traceInfo":{},"minConsumingFreshnessTimeMs":0}
+{"resultTable":{"dataSchema":{"columnDataTypes":["DOUBLE","DOUBLE","LONG","LONG","DOUBLE","DOUBLE"],"columnNames":["percentile(longDimSV1, 80.0)","percentilemv(intDimMV1, 90.0)","percentileest(longDimSV1, 80.0)","percentileestmv(intDimMV1, 90.0)","percentiletdigest(longDimSV1, 80.0, 100.0)","percentiletdigestmv(intDimMV1, 90.0, 100.0)"]},"rows":[[7611.0,462.0,7611,462,7611.0,462.0]]},"exceptions":[],"numServersQueried":1,"numServersResponded":1,"numSegmentsQueried":1,"numSegmentsProcessed":1,"numSegmentsMatched":1,"numDocsScanned":10,"numEntriesScannedPostFilter":20,"numGroupsLimitReached":false,"totalDocs":10,"timeUsedMs":8,"segmentStatistics":[],"traceInfo":{},"minConsumingFreshnessTimeMs":0}
```

Comment:
If the 3rd argument (comp factor) is optional and default behavior is same then the existing test results don't need to change and we can add new expected result rows ?

Reply:
I've modified the `getResultColumnName()` to always print the compression factor for v1 type queries. Do you recommend not printing that unless a non-default is used?

I can add additional result rows for the new syntax

Reply:
+1 on adding additional result rows for the new syntax

Reply:
Yes I don't think result column name should be changed and we should try to avoid changing existing expected output and instead add net new tests / expected rows

Reply:
```
2023/04/19 22:48:29.807 ERROR [SqlResultComparator] [main] Got exception: [{"errorCode":200,"message":"QueryExecutionError:\norg.apache.pinot.spi.exception.BadQueryRequestException: Invalid aggregation function: percentiletdigest(longDimSV1,'80','1000')\n\tat org.apache.pinot.core.query.aggregation.function.AggregationFunctionFactory.getAggregationFunction(AggregationFunctionFactory.java:240)\n\tat org.apache.pinot.core.query.request.context.QueryContext$Builder.generateAggregationFunctions(QueryContext.java:457)\n\tat org.apache.pinot.core.query.request.context.QueryContext$Builder.build(QueryContext.java:433)\n\tat org.apache.pinot.core.query.request.context.utils.BrokerRequestToQueryContextConverter.convertSQL(BrokerRequestToQueryContextConverter.java:150)"}] when querying!
```

@jasperjiaguo suggested removing the new query pattern from the compatibility testing file for now since the older pattern isn't supported on older versions. Will add these tests later on